lib/checks/canMultiplyLikeTermsNthRoots.js

```diff
@@ -0,0 +1,19 @@
+const Node = require('../node');
+
+function canMultiplyNthRoots(node) {
+  // checks if node is a multiplication of nthRoot nodes
+  // all the terms has to have the same root node to be multiplied
+
+  if (!Node.Type.isOperator(node, '*')
+      || !(node.args.every(term => Node.Type.isFunction(term, 'nthRoot')))){
```

Comment:
hmm what happens if you have `nthRoot(x, 2) * nthRoot(x, 2) * 3` ? It won't get combined here right?

Is that handled in collect and combine? If so, add a test for this case (I don't think I see one) -
 if not maybe you can make a separate change to collect them there so they can be combined with this function